src/main/java/org/quiltmc/loader/impl/discovery/ClasspathModCandidateFinder.java

```diff
@@ -173,7 +173,7 @@ public static Path getLoaderPath() {
 
 	public static Path getGameProviderPath() {
 		try {
-			return UrlUtil.asPath(QuiltLoaderImpl.INSTANCE.getGameProvider().getClass().getProtectionDomain().getCodeSource().getLocation());
+			return UrlUtil.asPath(QuiltLoaderImpl.class.getProtectionDomain().getCodeSource().getLocation());
```

Comment:
This change looks a bit weird - this method is intended to find the game provider jar to load plugins from it - does that functionality not work for you? (Or do you set the launcher so `getLoaderPath` actually returns the game provider path by mistake?)

(This method is used next to `getLoaderPath()`, so it should probably not return the same location)